src/services

```diff
@@ -14,7 +14,7 @@ p({"class":"foam.nanos.boot.NSpec", "name":"http",                             "
 p({"class":"foam.nanos.boot.NSpec", "name":"httprouter",                       "lazy":true,  "serve":false, "serviceClass":"foam.nanos.http.NanoRouter"})
 p({"class":"foam.nanos.boot.NSpec", "name":"websockets",                       "lazy":false, "serve":false, "serviceClass":"foam.nanos.ws.NanoWebSocketServer"})
 p({"class":"foam.nanos.boot.NSpec", "name":"pmLogger",                         "lazy":true,  "serve":false, "serviceClass":"foam.nanos.pm.DAOPMLogger"})
-p({"class":"foam.nanos.boot.NSpec", "name":"auth",                             "lazy":true,  "serve":true,  "authenticate": false, "boxClass":"foam.nanos.auth.AuthServiceSkeleton", "serviceScript":"return new foam.nanos.auth.twofactor.TwoFactorAuthService.Builder(x).setDelegate(new foam.nanos.auth.PasswordExpiryAuthService.Builder(x).setDelegate(new foam.nanos.auth.EnabledCheckAuthService.Builder(x).setDelegate(new foam.nanos.auth.CachingAuthService(new foam.nanos.auth.UserAndGroupAuthService(x))).build()).build()).build();","client":"{\"class\":\"foam.nanos.auth.CachedAuthServiceProxy\",\"delegate\":{\"class\":\"foam.nanos.auth.ClientLoginAuthService\"}}"})
+p({"class":"foam.nanos.boot.NSpec", "name":"auth",                             "lazy":true,  "serve":true,  "authenticate": false, "boxClass":"foam.nanos.auth.AuthServiceSkeleton", "serviceScript":"return new foam.nanos.auth.SystemAuthService.Builder(x).setDelegate(new foam.nanos.auth.twofactor.TwoFactorAuthService.Builder(x).setDelegate(new foam.nanos.auth.PasswordExpiryAuthService.Builder(x).setDelegate(new foam.nanos.auth.EnabledCheckAuthService.Builder(x).setDelegate(new foam.nanos.auth.CachingAuthService(new foam.nanos.auth.UserAndGroupAuthService(x))).build()).build()).build()).build();","client":"{\"class\":\"foam.nanos.auth.CachedAuthServiceProxy\",\"delegate\":{\"class\":\"foam.nanos.auth.ClientLoginAuthService\"}}"})
```

Comment:
Please consider converting to the new multi-line format for journal files with scripts in them. It makes it much easier to read and review. See an example here: https://github.com/foam-framework/foam2/blob/master/src/services#L42-L60.

Reply:
converted